src/scripts/Autosplitter.gd

```diff
@@ -0,0 +1,59 @@
+extends Node
+
+
+const PORT := 5678
+
+var server: WebSocketServer
+var peers := []
+
+
+func _ready() -> void:
```

Comment:
We should probably make this an option, just in case people don't want the socket running by default. You could also maybe (idk how feasible this is) make the port number a text box entry.

Reply:
Good idea, I'll do that.

Number text box should be very easy :)

Reply:
A toggle and a number box have been added to the options menu.